### DISCUSSION TITLE
Installing to existing partitions

### DISCUSSION THREAD

**Discussion by grg3** [archlinux/archinstall#194] [Q&A]
I tried installing to existing partitions with guided script, but could not figure out how to pass partition mount correctly to the script. Is it possible to provide an example of install with existing partitions?

**Comment by Torxed** [comment]
On it!

**Reply by theosmith0308** [reply]
I am also having a trouble installing to an existing partition. Especially btrfs and when trying to create additional sub volumes it just fails.
